treeherder/extract/extract_jobs.py

```diff
@@ -116,8 +116,20 @@ def extract(self, settings, force, restart, start, merge):
                 # SOME LIMITS PLACES ON STRING SIZE
                 for fl in jx.drill(acc, "job_log.failure_line"):
                     fl.message = strings.limit(fl.message, 10000)
-                for r in acc:
-                    r.etl.timestamp = Date.now()
+                # LIMIT NUMBER OF text_log_error
+                for job in acc:
+                    if len(job.text_log_error) > 100:
+                        shorter = []
+                        known = set()
+                        for e in job.text_log_error:
+                            if e.line in known:
+                                continue
+                            known.add(e.line)
+                            shorter.append(e)
+                        job.text_log_error = shorter[:100]
+                # ASSIGN TIMESTAMP
+                for job in acc:
+                    job.etl.timestamp = Date.now()
```

Comment:
The original problem was the `text_log_error` records were too numerous for BigQuery.  CHOP! CHOP!

Reply:
Sarah's code got backed out yesterday because a limit had been removed. The limit is being re-added and being reviewed.
It's good to have this either way.